NeosModLoader/NeosModLoader.csproj

```diff
@@ -4,14 +4,18 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NeosModLoader</RootNamespace>
-    <AssemblyTitle>NeosModLoader</AssemblyTitle>
+    <Headless Condition="'$(Headless)'==''">false</Headless>

```

Comment:
Since later on we're testing only against true and false (and ignoring the possibility that it might be "0" or "1" for example), would probably be a good to add a definition after this to set it to true if it is not exactly "false" to make sure that one of the two cases always matches?

Reply:
Fixed in 942b72ce